parm/soca/obs/obs_list.yaml

```diff
@@ -15,6 +15,7 @@ observers:
 # Ice concentration
 - !INC ${OBS_YAML_DIR}/icec_amsr2_north.yaml
 - !INC ${OBS_YAML_DIR}/icec_amsr2_south.yaml
+- !INC ${OBS_YAML_DIR}/icec_mirs.yaml
```

Comment:
could you comment that out @apchoiCMD , I don't think we want this to be part of the default obs list setup yet.